src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css

```
@@ -0,0 +1,180 @@
+/*Contains variables for colors and fonts used in the theme*/
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap");
+
+@font-face {
+  font-family: "Source Sans Pro Light";
+  src: url(../fonts/SourceSansPro-Light.ttf);
+}
+
+@font-face {
+  font-family: "Source Sans Pro";
+  src: url(../fonts/SourceSansPro-Regular.ttf);
+}
+
+:root {
+  /* Ansys specific changes to the theme */
+
+  /*****************************************************************************
+    * Ansys Font family
+    **/
```

What about using Javadoc style for these comments? This is a common standard understood by tools like Doxygen:

```suggestion
  /**
    * Ansys Font family
    */
```